### PR TITLE
global: use date from inspire-utils

### PR DESCRIPTION
**Pull request by jacquerie**

### CODE REVIEW

File: inspire_dojson/common/rules.py [hunk 1]
```diff
@@ -322,7 +323,7 @@ def legacy_creation_date(self, key, value):
     if 'legacy_creation_date' in self:
         return self['legacy_creation_date']
 
-    return value.get('x')
+    return normalize_date(value.get('x'))
```
Comment (michamos) [inline inspire_dojson/common/rules.py]: this doesn't work with `2017-10-00`/`2017-00-00`. You need to strip trailing `-00` first.

Comment (jacquerie) [reply inspire_dojson/common/rules.py]: We have those in `961__x`? Can I have an example?

Comment (michamos) [reply inspire_dojson/common/rules.py]: oh, sorry, this is `961__x`. I saw the `x` and thought about `111__x` in conferences.

Comment (michamos) [reply inspire_dojson/common/rules.py]: https://inspirehep.net/record/1364816

Comment (jacquerie) [reply inspire_dojson/common/rules.py]: Ah, yes, sure, but that is handled in this other part of the code: https://github.com/inspirehep/inspire-dojson/blob/41b9fdd8ab91e8c3a49b8d2472578f8b9b464f3b/inspire_dojson/conferences/rules.py#L36-L46

File: inspire_dojson/hep/rules/bd2xx.py [hunk 2]
```diff
@@ -129,9 +130,7 @@ def imprints2marc(self, key, value):
 
 @hep.over('preprint_date', '^269..')
 def preprint_date(self, key, value):
-    if isinstance(value, (tuple, list)):
-        return min(elem['c'] for elem in value if 'c' in elem)
-    return value.get('c')
+    return normalize_date(value.get('c'))
```
Comment (jacquerie) [inline inspire_dojson/hep/rules/bd2xx.py]: This code was dealing with multiple `269__c` in the same `269` for no reason: there aren't any.

**Pull request merged**